cea/technologies/thermal_network/thermal_network_costs.py

```diff
@@ -0,0 +1,563 @@
+            for building in optimal_network.building_names:
+                assert optimal_network.locator.get_optimization_disconnected_folder_building_result_heating(building), "Missing diconnected building files. Please run disconnected_buildings_heating first."
+            # Read in disconnected cost of all buildings
+                disconnected_cost = optimal_network.locator.get_optimization_disconnected_folder_building_result_heating(building)
+    '''
+    if network_info.network_type == 'DC':
+        supplied_systems = []
+        # iterate through all possible cooling systems
+        for system in network_info.full_cooling_systems:
+            if system not in network_info.config.thermal_network.substation_cooling_systems:
+                # add system to list of loads that are supplied at building level
+                disconnected_systems.append(system)
+        if len(disconnected_systems) > 0:
+            # check if we have any disconnected systems
+            system_string = find_cooling_systems_string(disconnected_systems)
+            # iterate through all buildings
+            for building_index, building in enumerate(network_info.building_names):
+                Opex_var_chiller = 0.0
+                Capex_chiller = 0.0
+                Opex_var_CT = 0.0
+                if building_index not in network_info.disconnected_buildings_index:
+                    # if this building is disconnected it will be calculated separately
+                    # Read in building demand
+                    disconnected_building_demand = pd.read_csv(
+                        network_info.locator.get_demand_results_file(building))
+                    if not system_string:
+                        # this means there are no disconnected loads. Shouldn't happen but is a failsafe
+                        peak_demand_kW = 0.0
+                        disconnected_demand_t_sum = 0.0
+                    else:
+                        for system_index, system in enumerate(system_string):  # iterate through all disconnected loads
+                            # go through all systems and sum up demand values
+                            if system_index == 0:
+                                disconnected_demand_t = disconnected_building_demand[system]
+                            else:
+                                disconnected_demand_t = disconnected_demand_t + disconnected_building_demand[system]
+                        peak_demand_kW = disconnected_demand_t.abs().max()  # calculate peak demand of all disconnected systems
+                        disconnected_demand_t_sum = disconnected_demand_t.abs().sum()
+                    print 'Calculate cost of disconnected loads in building ', building
+                    if network_info.config.thermal_network_optimization.yearly_cost_calculations:
+                        COP_chiller_system = VCCModel.calc_VCC_COP(network_info.config, system_string,
```

Comment:
please confirm this is correctly implemented (see FIXME in line 245)

Reply:
Good catch, now it is correct